src/containers/UsernameTranslation.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+
+/*
```

Comment:
I would expect broader information about this component in the `docs`, not in the code itself. What I mean is this:
```
At the moment works ONLY with apiPath='seacat_auth'
- userId (required, array) - user ids to be translated
- apiPath - default (and only working so far) 'seacat_auth'
- cleanupTime - default set to 24 hours
```
It should be a markdown file, so it should keep the markdown syntax then.

Reply:
Remark: We don't have "user ids", we have "credentials_ids" ... @byewokko